blog/2020-10-09-hs-to-coq/2020-10-09-hs-to-coq.md

```diff
@@ -0,0 +1,1172 @@
+---
+title: "Tutorial: Verify Haskell Programs with hs-to-coq"
+author: Li-yao Xia
+description: A tutorial on hs-to-coq
+tags: haskell, coq
+withtoc: true
+alectryon: [-Q, "blog/2020-10-09-hs-to-coq/lib/Src", Src, -Q, "blog/2020-10-09-hs-to-coq/lib/Proofs", Proofs]
+alectryon-cache: "blog/2020-10-09-hs-to-coq/alectryon/"
+---
+
+## Preface
```

Comment:
The whole preface section is a little grandiose, which is cool, but I think it influenced your sentence structure a little too much. I'll put a few specific comments in further down, but I think you should just review the whole section a little.

Sorry if any of this comes across as harsh, I really don't mean it that way.

Reply:
I also find this section a little too over the top so I appreciate feedback about it!

I would assume most readers are familiar with the background already, so I cranked it up to be more thought-provoking (even if in disagreement) and more exciting for the occasional newcomer who's somehow never heard about any of this.
That's why it's more of a "Preface" than an "Introduction".